lx_annotate/settings.py

```
@@ -13,7 +13,19 @@
 from pathlib import Path
 import os
 from dotenv import load_dotenv
```

Ruff (0.8.2)

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)